capstone/src/main/webapp/cover.css

```diff
@@ -0,0 +1,107 @@
+/*
+ * Globals
+ */
```

Comment:
I think the correct convention is to have all of this on one line. I see some of the comments are already like that. If it is, we should make it all on one line everywhere

Reply:
addressed in "address nits"